modules/social_features/social_core/src/Plugin/Validation/Constraint/ExternalIdentifierEmptySubfieldsConstraint.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\social_core\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Provides a custom constraint for fields.
+ *
+ * @Constraint(
+ *   id = "ExternalIdentifierEmptySubfieldsConstraint",
+ *   label = @Translation("Makes sure that all required subfields values are set", context = "Validation"),
+ *   type = {"field"}
+ * )
+ */
+class ExternalIdentifierEmptySubfieldsConstraint extends Constraint {
+
+  /**
+   * The error message when all required subfields values not are set.
```

Comment:
Small grammar suggestion: 

`The error message when all required subfield values are not set`